src/LightSpeed/toolbox.js

```diff
@@ -67,4 +82,32 @@ var LSToolbox = GObject.registerClass(class LSToolbox extends Toolbox {
         this._combinedTopic.reset();
         // The user functions are reset by the LightSpeed app already
     }
+
+    async _showTopicsInitially() {
+        const topics = ['spawnEnemy', 'updateAsteroid', 'updateSpinner',
```

Comment:
Should 'game' be included here?

Reply:
Yeah, good question. I originally didn't include it because that topic is always shown unconditionally, and adding it here would require us to set its corresponding game state key somewhere, otherwise it wouldn't show up. However, that does make it so that we couldn't hide that topic in the future unless we made that change here.

Your call as the reviewer, I guess :smile: I can implement it either way.

Reply:
I see! Thanks for the explanation. In that case I'd leave the refactor until it's necessary.